src/components/sidebar/developDocsSidebar.tsx

```diff
@@ -37,10 +37,10 @@ export function DevelopDocsSidebar({
     return toTree(apiNodes);
   };
   return (
-    <aside className={styles.sidebar}>
+    <aside className={`${styles.sidebar} py-3`}>
```

Comment:
can this not just be

```suggestion
    <aside className={`${styles.sidebar} p-3`}>
```

or does this need to be spread out like this?

Reply:
I simplified it by just moving the padding altogether to the `aside` but a a horizontal padding of `6` would be too much